docs/book/getting_started/existing_providers.md

```diff
@@ -5,5 +5,4 @@ There are multiple implementations of the Cluster API for different environments
 These implementations are called _providers_. To find a partial list of the 
 existing providers see the Cluster API [Provider Implementations List][provider-implementations-list]
 
-[provider-implementations-list]: https://github.com/kubernetes-sigs/cluster-api/blob/master/README.md#provider-implementations
-.
+[provider-implementations-list]: https://github.com/kubernetes-sigs/cluster-api/blob/master/README.md#provider-implementations.
```

Comment:
This is a definition. The missing period is in the line above where the link is referenced.